### ISSUE TITLE
Bug: "perfectionist/sort-modules": Cannot read properties of undefined (reading 'type')

### ISSUE THREAD

**Issue by kylsgl** [azat-io/eslint-plugin-perfectionist#382]
### Describe the bug

"perfectionist/sort-modules" Error after upgrading to `4.0.2` from `3.9.1`
```
Oops! Something went wrong! :(

ESLint: 8.57.1

TypeError: Cannot read properties of undefined (reading 'type')
Occurred while linting E:\app\src\components\common\segmented-control\enums.ts:2
Rule: "perfectionist/sort-modules"
    at extractDependencies (E:\app\node_modules\.pnpm\eslint-plugin-perfectionist@4.0.2_eslint@8.57.1_typescript@5.6.3\node_modules\eslint-plugin-perfectionist\dist\rules\sort-modules.js:443:16)
    at parseNode (E:\app\node_modules\.pnpm\eslint-plugin-perfectionist@4.0.2_eslint@8.57.1_typescript@5.6.3\node_modules\eslint-plugin-perfectionist\dist\rules\sort-modules.js:242:16)
    at parseNode (E:\app\node_modules\.pnpm\eslint-plugin-perfectionist@4.0.2_eslint@8.57.1_typescript@5.6.3\node_modules\eslint-plugin-perfectionist\dist\rules\sort-modules.js:197:13)
    at analyzeModule (E:\app\node_modules\.pnpm\eslint-plugin-perfectionist@4.0.2_eslint@8.57.1_typescript@5.6.3\node_modules\eslint-plugin-perfectionist\dist\rules\sort-modules.js:271:5)
    at Program (E:\app\node_modules\.pnpm\eslint-plugin-perfectionist@4.0.2_eslint@8.57.1_typescript@5.6.3\node_modules\eslint-plugin-perfectionist\dist\rules\sort-modules.js:157:9)
    at ruleErrorHandler (E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\linter\linter.js:1076:28)
    at E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\linter\safe-emitter.js:45:58
    at Array.forEach (<anonymous>)
    at Object.emit (E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\linter\safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\linter\node-event-generator.js:297:26)
    at NodeEventGenerator.applySelectors (E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\linter\node-event-generator.js:326:22)
    at NodeEventGenerator.enterNode (E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\linter\node-event-generator.js:340:14)
    at CodePathAnalyzer.enterNode (E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\linter\code-path-analysis\code-path-analyzer.js:803:23)
    at E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\linter\linter.js:1111:32
    at Array.forEach (<anonymous>)
    at runRules (E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\linter\linter.js:1106:15)
    at Linter._verifyWithoutProcessors (E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\linter\linter.js:1355:31)
    at Linter._verifyWithConfigArray (E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\linter\linter.js:1807:21)
    at Linter.verify (E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\linter\linter.js:1437:65)
    at Linter.verifyAndFix (E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\linter\linter.js:2068:29)
    at verifyText (E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\cli-engine\cli-engine.js:254:48)
    at CLIEngine.executeOnFiles (E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\cli-engine\cli-engine.js:834:28)
    at ESLint.lintFiles (E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\eslint\eslint.js:551:23)
    at Object.execute (E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\lib\cli.js:421:36)
    at async main (E:\app\node_modules\.pnpm\eslint@8.57.1\node_modules\eslint\bin\eslint.js:152:22)
```

### Code example

enums.ts
```ts
// eslint-disable-next-line import/prefer-default-export
export enum Marker {
	DURATION = '--marker-duration',
	HEIGHT = '--marker-height',
	TRANSFORM = '--marker-transform',
	WIDTH = '--marker-width',
}
```

### ESLint version

8.57.1

### ESLint Plugin Perfectionist version

4.0.2

### Validations

- [X] Read the [docs](https://perfectionist.dev).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by hugop95** [comment]
Looking into it.

**Comment by azat-io** [comment]
Fixed in 979ece80fc95d8a80494bb95de6414853e744506.
Released in [v4.0.3](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v4.0.3).
